### PR TITLE
Completed API Model & Validations

### PR DESCRIPTION
**Pull request by Sakaa4Short**

### CODE REVIEW

**Review by elysemontano — CHANGES_REQUESTED** [review]
I am a little confused on what this PR should contain. I am seeing controller methods and request specs which don't belong with validations.

**Comment by Sakaa4Short** [comment]
Removed Validations so that only controller method and request specs are apparent.

**Review by elysemontano — CHANGES_REQUESTED** [review]
This PR branch and Trello card states that this should be for validations, not for API endpoints which is the work I am seeing.

File: spec/requests/apartments_spec.rb [hunk 1]
```diff
@@ -1,7 +1,35 @@
...
+      )
+      get '/apartments'
+
+      apartment = JSON.parse(response.body)
+      expect(response).to have_http_status(200)
+      expect(apartment.first['street']).to eq('4 Privet Drive')
+    end
   end
+
+  # test for creating a new apartment will live here
```
Comment (elysemontano) [inline spec/requests/apartments_spec.rb]: Let's remove this from this PR.

File: app/controllers/apartments_controller.rb [hunk 2]
```diff
@@ -1,2 +1,21 @@
...
+        render json: apartment
+      else
+        render json: apartment.errors, status: 422
+      end
+    end
+  
+    private
+    def apartment_params
+      params.require(:apartment).permit(:street, :city, :state, :manager, :email, :price, :bedrooms, :bathrooms, :pets, :image, :user_id)
+    end
```
Comment (elysemontano) [inline app/controllers/apartments_controller.rb]: Let's remove this from this PR.

**Review by elysemontano — CHANGES_REQUESTED** [review]
Still seeing endpoint content on this PR. Please remove the blocks of code I highlighted in apartments controller and request specs

File: app/controllers/apartments_controller.rb (hunk 2 above)
Comment (Sakaa4Short) [reply app/controllers/apartments_controller.rb]: Lines of code removed

File: spec/requests/apartments_spec.rb (hunk 1 above)
Comment (Sakaa4Short) [reply spec/requests/apartments_spec.rb]: Lines of code removed

**Review by elysemontano — APPROVED** [review]

**Pull request merged**